<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_data</class>
 <widget class="QMainWindow" name="User_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>753</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>753</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>753</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User data - Cataloging Department</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Эмблема2.png</normaloff>Эмблема2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label__udata_phone">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label__udata_bth">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label__udata_registr">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>660</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_name">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 137, 137);
color: rgb(229, 228, 224);</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label__udata_lname">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label__udata_name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>759</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>801</width>
      <height>759</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>759</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>data1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lab_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(156, 128, 124);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> По всем вопросам - обращайтесь к библиотекарю.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label__udata_address">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(86, 54, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label__udata_phone</zorder>
   <zorder>label__udata_bth</zorder>
   <zorder>label__udata_registr</zorder>
   <zorder>label_user_name</zorder>
   <zorder>btn_back</zorder>
   <zorder>label__udata_lname</zorder>
   <zorder>label__udata_name</zorder>
   <zorder>lab_info</zorder>
   <zorder>label__udata_address</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Курсовая.qrc"/>
 </resources>
 <connections/>
</ui>
